Makefile

```diff
@@ -124,6 +124,10 @@ publish_to_maven_local:
 publish_to_artifactory:
 	$(docker_command) ./gradlew $(project) $(log_level) $(params) publishToArtifactory -PprojectVersion=$(version) -Dorg.gradle.internal.publish.checksums.insecure=true
 
+stage_ui_tests:
```

Comment:
What is `stage`?

Reply:
i wanted to express that it is ongoing changes, staged in terms of git, but not released yet